flatisfy/web/js_src/components/flatstableline.vue

```diff
@@ -11,7 +11,7 @@
                     <i class="fa fa-star" aria-hidden="true" :title="capitalizedStatus"></i>
                 </template>
 
-                [{{ flat.id.split("@")[1] }}] {{ flat.title }}
+                [{{ flat.id.split("@")[1] }}] {{ flat.title }} {{ flat.is_expired ? "EXPIRED" : null }}
```

Comment:
I'd actually rather prepend it, something like

```
[{{ flat.id.split("@")[1] }}] {{ flat.is_expired ? "[EXPIRED]" : null }} {{ flat.title }}
```

What do you think about it?